content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +85,234 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
+
+{{n.next()}}. Run unit tests built into the server executable. This could take about 5 minutes, depending on your hardware specs. (optional, but recommended) <!--{# ***TODO: We should provide info about what to do if you get failures. What should the user do? PM looking into this.*** #}-->
+
+        ./rippled --unittest
+
+{{n.next()}}. `rippled` requires the `rippled.cfg` configuration file to run. You can find an example config file, `rippled-example.cfg` in `rippled/cfg`. Make a copy and save it as `rippled.cfg` in a location that enables you to run `rippled` as a non-root user. Access the `rippled` directory and run:
+
+        mkdir -p /etc/opt/ripple ***TODO: changed this from ~/.config/ripple to match guidance coming in conf-file-location.md PR -- okay?***
+        cp cfg/rippled-example.cfg /etc/opt/ripple/rippled.cfg
+
+{{n.next()}}. Edit `rippled.cfg` to set necessary file paths. The user you plan to run `rippled` as must have write permissions to all of the paths you specify here. ***TODO: We need to add this step to the centos and ubuntu tasks, correct? For centos and ubuntu, the config files are automatically placed in the correct directories, but the following paths still need to be manually checked/updated to ensure that there are no permission issues, correct?***
+
+      * Set the `[node_db]` path to the location where you want to store the ledger database.
+
+      * Set the `[database_path]` to the location where you want to store other database data. (This includes an SQLite database with configuration data, and is typically one level above the `[node_db]` path field.)
+
+      * Set the `[debug_logfile]` to a path where `rippled` can write logging information.
+
+      These are the only configurations required for `rippled` to start up successfully. All other configuration is optional and can be tweaked after you have a working server. For more information, see [Additional Configurations](#additional-configuration).
+
+{{n.next()}}. `rippled` requires the `validators.txt` file to run. You can find an example validators file, `validators-example.txt`, in `rippled/cfg/`. Make a copy and save it as `validators.txt` in the same folder as your `rippled.cfg` file. Access the `rippled` directory and run:
+
+        cp cfg/validators-example.txt /etc/opt/ripple ***TODO: changed this from ~/.config/ripple to match guidance coming in conf-file-location.md PR -- okay?***
+
+      **Warning:** Ripple has designed a decentralization plan with maximum safety in mind. During the transition, you should not modify the `validators.txt` file except as recommended by Ripple. Even minor modifications to your validator settings could cause your server to diverge from the rest of the network and report out of date, incomplete, or inaccurate data. Acting on such data can cause you to lose money.
+
+{{n.next()}}. Access your build directory, `my_build` for example, and start the `rippled` service.
+
+        sudo ./rippled
```

Comment:
This runs `rippled` as root—not recommended. If you set up file paths for your config and databases to directories you can write, then `rippled` should run without root permissions.

Caution: if you've ever started rippled as root (including using `sudo`) then it probably created some database files as a root user, which means it'll fail with a permissions error if you try to run it as non-root user, _even if the user has permissions to the directories containing the databases_. To fix this, you have to (using root permissions) delete `rippled`'s database files (in particular, the ones it created when it ran as an admin user, but also possibly ones it created when run as a non-root user before that, because they may be incomplete/corrupt if you delete some database files). The easiest way to do this is `sudo rm -r` the `rippled` database directory (assuming you don't have anything else in that directory) and then re-create it without sudo / admin, as the user you intend to run `rippled` with.

Reply:
Do you have a 🔮 in which you can see what I'm doing on my command line? 😂  Deleted my db files and it ran without sudo. Removed `sudo` here.